cloud-aws/src/test/java/com/sequenceiq/cloudbreak/cloud/aws/view/AwsRdsDbParameterGroupViewTest.java

```diff
@@ -48,7 +50,7 @@ void getDBParameterGroupFamilyTestWhenPgSqlAndNullVersion() {
 
         AwsRdsDbParameterGroupView underTest = new AwsRdsDbParameterGroupView(databaseServer);
 
-        assertThatCode(underTest::getDBParameterGroupFamily).isInstanceOf(NullPointerException.class);
+        assertThat(underTest.getDBParameterGroupFamily()).isEqualTo(POSTGRES_10);
```

Comment:
I would extend the parametrized tests below that value "10" is parsed to "10" and "postgres10".

Reply:
Ok